templates/js/api-plugin-from-scratch-bearer/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
```

Comment:
tsconfig for js template?

Reply:
Oh, I copied from the ts template, removed